docassemble/ALWeaver/custom_values.py

```diff
@@ -178,6 +180,63 @@ def get_yml_deps_from_choices(choices: Union[List[str], DADict]):
         return choices
 
 
+def get_full_dep_details(dep_category=None):
+    """Get the full dictionary describing each dependency (including custom ones) so that it can be
+    filtered and used as needed."""
+    dep_choices = []
+
+    # TODO: do we want to prefix the choice with the package name?
+    for capability in _al_weaver_capabilities:
+        if dep_category == "organization":
+            dep_choices.extend(
+                _al_weaver_capabilities[capability].get("organization_choices", [])
+            )
+        elif dep_category == "jurisdiction":
+            dep_choices.extend(
+                _al_weaver_capabilities[capability].get("jurisdiction_choices", [])
+            )
+
+    return list(unique_everseen(dep_choices))
+
+
+def get_matching_deps(dep_category=None, state=None):
+    """Get the dependencies that match the specified state.
+    State "ANY" has a special meaning."""
+    dep_choices = []
+
+    # TODO: do we want to prefix the choice with the package name?
+    for capability in _al_weaver_capabilities:
```

Comment:
Minor: if you do this:
```suggestion
    for capability in _al_weaver_capabilities.values():
```

Then the list comprehensions below get a little shorter, which collapses the `for` loop to a single line. IMO it makes is a bit more readable. At first glance right now it looks like the list comprehension is it's own inline function or something crazy.

I hope I eventually get used to the weird black formatting choices, lol.